uint32 poseArrayIndex
geometry_msgs/Pose poseArrayPose
geometry_msgs/Pose pose
string objClass
float32 objProb
bool isReported
uint32 imageId
string vehicleReporter
